lib/node_modules/@stdlib/stats/base/dists/geometric/quantile/README.md

```diff
@@ -145,6 +145,105 @@ for ( i = 0; i < 10; i++ ) {
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/stats/base/dists/geometric/quantile.h"
+```
+
+#### stdlib_base_dists_geometric_quantile( p, p_success )
```

Comment:
It would be better to use the same parameter names as we do for the JS implementation.

As-is, this also doesn't match the description in the following line.